punishments.js

```diff
@@ -1098,6 +1098,7 @@ Punishments.checkLockExpiration = function (userid) {
 	const punishment = Punishments.userids.get(userid);
 
 	if (punishment) {
+		if (Users(userid) && Users(userid).permalocked) return ` (never expires; you are permalocked)`;
```

Comment:
Do not repeatedly use accessors like `Users(userid)`. If you need it more than once, make a variable for it.